src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-SideNavigation/Common/types.ts

```diff
@@ -5,7 +5,7 @@ export interface LayerProps {
   marginBottom?: boolean;
   prepend?: React.ReactNode;
   append?: React.ReactNode;
-  [rest: string]: any;
+  [rest: number]: string;
```

Comment:
Can you please explain this one?

Reply:
@Marvin9 I referred to this approach: https://stackoverflow.com/a/47039640
Also, I couldn't find any more types that would fit correctly without raising any additional errors / wranings.

Reply:
What your approach says is "If any props other than defined ones have type of number than their value should be string". which makes no sense because you cannot do ```<CommonLayer 6="foo" />```. 

Try yourself, pass onClick event to ```CommonLayer```. Now change type to previous ones and it will solve issue. 

https://stackoverflow.com/a/58201122

What you refer was irrelevant.

Reply:
Ohh I see what you are saying. My bad I didn't notice it. Thanks, I'll revert this change.

Reply:
@Marvin9 reverted the change.